services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if not genres_ids and not actors_ids:
+        queryset = queryset
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids and not actors_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids and not genres_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    movie = Movie.objects.get(id=movie_id)
+    return movie

```

Comment:
There is no need to create `movie` variable here, remove it. Make corresponding changes in the other places as well